resources/assets/js/components/Conference.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="conference">
+        <friends-list :list="newFriends" key="new-friends" :conference-id="conferenceId" descriptor="New"></friends-list>
+
+        <friends-list :list="onlineFriends" key="online-friends" :conference-id="conferenceId" descriptor="Online"></friends-list>
+    </div>
+</template>
+
+<script>
+    export default {
+        data: function() {
+            return {
+                currentUserId: Confomo.userId,
+                conferenceId: Confomo.conferenceId,
+
+                newFriends: [],
+                onlineFriends: []
+            };
+        },
+
+        components: {
+            FriendsList: require('./FriendsList.vue')
```

Comment:
i've been wondering how we should handle inline requiring as we move into ES6 style import. Because, right now, we're mixing them. But I don't know how you you should import a .vue component inside another using the ES6 notation. Let's keep this for now but hopefully we figure out what the actual best practice is there.